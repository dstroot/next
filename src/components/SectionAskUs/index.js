import React from 'react';
import Form from '../Form';

const SectionAskUs = () => {
  return (
    <div className="nxt_bg-gray py-6">
      <div className="container text-center">
        <h1 className="nxt_heading-large mb-5">Ask us anything</h1>
        <div className="row">
          <div className="col-md-6">
            <Form />
          </div>
          <div className="col-md-6 text-left">
            <h2>Next by Pacific Life</h2>
            <a href="mailto: hello@meetnext.com">hello@meetnext.com</a>
            <p>(833) 646-NEXT (6398) </p>
          </div>
        </div>
      </div>
    </div>
  );
};

export default SectionAskUs;
