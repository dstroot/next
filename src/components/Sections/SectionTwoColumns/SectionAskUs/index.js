import React from 'react';

//import components
import Form from '../../../Form';
import SectionTwoColumns from '../../SectionTwoColumns';

const SectionAskUs = () => {
  return (
    <SectionTwoColumns
      styleBackground="bg-light"
      styleTextAlign="text-center"
      sectionHeadStyle="nxt_heading-large"
      sectionHead="Ask us anything"
      styleLeft="order-2 mt-5 text-left"
      contentLeft={
        <div>
          <h2>Next by Pacific Life</h2>
          <a href="mailto: hello@meetnext.com">hello@meetnext.com</a>
          <p>(833) 646-NEXT (6398) </p>
        </div>
      }
      styleRight="order-1"
      styleRight="text-left"
      contentRight={<Form />}
    />
  );
};

export default SectionAskUs;

{
  /* <div className="nxt_bg-gray py-6">
      <div className="container text-center">
        <h1 className="nxt_heading-large mb-5">Ask us anything</h1>
        <div className="row">
          <div className="col-md-6">
            <Form />
          </div>
          <div className="col-md-6 text-left mt-4">
            <h2>Next by Pacific Life</h2>
            <a href="mailto: hello@meetnext.com">hello@meetnext.com</a>
            <p>(833) 646-NEXT (6398) </p>
          </div>
        </div>
      </div>
    </div> */
}
