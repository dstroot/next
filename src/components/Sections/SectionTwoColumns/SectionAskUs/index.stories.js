import React from 'react';
import { storiesOf } from '@storybook/react';
import SectionAskUs from '../SectionAskUs';

storiesOf('NEXT|sections|SectionAskUs', module)
  .addParameters({
    info: {
      inline: true,
      header: false,
      text: `This supports markdown!
      
      ~~~js
      console.log("hello");
      ~~~
      `,
    },
  })
  .add('show SectionAskUs', () => <SectionAskUs />);
