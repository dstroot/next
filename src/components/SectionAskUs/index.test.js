import React from 'react';
import { create } from 'react-test-renderer';
import { mount } from 'enzyme';
import SectionAskUs from '../SectionAskUs';

describe('SectionAskUs', () => {
  // mount the component
  let mountedComponent;
  const getMountedComponent = () => {
    if (!mountedComponent) {
      mountedComponent = mount(<SectionAskUs />);
    }
    return mountedComponent;
  };

  beforeEach(() => {
    mountedComponent = undefined;
  });

  it('it should render', () => {
    let tree = create(<SectionAskUs />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
