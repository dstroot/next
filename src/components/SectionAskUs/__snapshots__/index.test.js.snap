// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionAskUs it should render 1`] = `
<div
  className="nxt_bg-gray py-6"
>
  <div
    className="container text-center"
  >
    <h1>
      Ask us anything
    </h1>
    <h2>
      Next by Pacific Life
    </h2>
    <p>
      hello@meetnext.com
    </p>
    <p>
      (833) 646-NEXT (6398) 
    </p>
  </div>
</div>
`;
